sdk/core/azure-core/src/main/java/com/azure/core/util/CoreUtils.java

```diff
@@ -161,6 +163,19 @@ public static boolean isNullOrEmpty(CharSequence charSequence) {
     }
 
 
+    /**
+     * Helper method that returns {@link UserAgentProperties} from properties defined in {@code propertiesFileName}.
+     *
+     * @param propertiesFileName The file name defining the properties.
+     * @return {@link UserAgentProperties}.
+     */
+    public static UserAgentProperties getUserAgentPropertiesFromProperties(String propertiesFileName) {
```

Comment:
Also is this name appropriate getUserAgentPropertiesFromProperties -> getUserAgentPropertiesFromFile ?

Reply:
It is actually property files. A particular file. 
Is getUserAgentProperties enough for the name. I don't like the name either.